manifests/install.pp

```diff
@@ -13,12 +13,11 @@
       $binary_group = 'Administrators'
```

Comment:
As designed, this PR will not take effect on windows installations.

Can you move these into params so that the too can be set via class params?